app/dashboard/static/js/app/view-tests-all.2018.9.js

```diff
@@ -28,251 +31,134 @@ require([
-        var results;
-
-        // Internally used to parse the results.
-        function _updateGroupsCount(result) {
-            var count;
-            var opId;
-
-            count = parseInt(result.result[0].count, 10);
-            opId = result.operation_id;
-            gGroupsCount[opId] = count;
-
-            updateOrStageCount(opId, count);
-        }
-
-        results = response.result;
-        if (results.length > 0) {
-            results.forEach(_updateGroupsCount);
-            if (!gDrawEventBound) {
-                gDrawEventBound = true;
-                gTestsTable.addDrawEvent(updateTestsTable);
-            }
-        } else {
-            html.replaceByClassTxt('groups-count-badge', '?');
-        }
-    }
-
-    function getGroupsCount(response) {
-        var batchOps;
-        var deferred;
 
-        function createBatchOp(value) {
-            var group = value.board;
-            var query = gBatchOpBase;
-
-            query += group;
+            /* Number of test case regressions */
             batchOps.push({
                 method: 'GET',
-                operation_id: 'groups-count-' + group,
+                operation_id: 'status-' + result._id.$oid,
                 resource: 'count',
-                document: 'test_group',
-                query: query
+                document: 'test_regression',
+                query: qStr,
             });
         }
 
-        if (response.length > 0) {
-            batchOps = [];
-            response.forEach(createBatchOp);
-
-            deferred = request.post(
-                '/_ajax/batch', JSON.stringify({batch: batchOps}));
+        batchOps = [];
+        results.forEach(createBatchOp);
+        deferred = request.post(
+            '/_ajax/batch', JSON.stringify({batch: batchOps}));
 
-            $.when(deferred)
-                .fail(error.error, getGroupsCountFail)
-                .done(getGroupsCountDone);
-        }
+        $.when(deferred)
+            .fail(error.error, getBatchStatusFailed)
+            .done(getBatchStatusDone)
     }
 
-    function getTestsDone(response) {
+    function updateTestTable(response) {
         var columns;
+        function _renderPlan(data, type, object) {
+            return ttest.renderKernel(data, type, 
+                '/test/job/'+object.job+
+                '/branch/'+object.git_branch+
+                '/kernel/'+object.kernel+
+                '/plan/'+data);
+        }
 
-        // Internal wrapper to provide the href.
-        function _renderDetails(data, type) {
-            return ttest.renderDetails('/test/board/' + data + '/', type);
+        function _renderTree(data, type) {
+            return ttest.renderTree(
+                data, type, '/job/' + data + '/');
         }
 
-        // Internal wrapper to provide the oreder count.
-        function _renderGroupsCount(data, type) {
-            var rendered;
+        function _renderKernel(data, type, object) {
```

Comment:
We should be doing the same thing as in this view, using the job table `renderKernel()` method:
https://github.com/kernelci/kernelci-frontend/blob/master/app/dashboard/static/js/app/view-jobs-job.2017.4.js#L378

Reply:
updated